### PR TITLE
update icon: npm (original, plain)

### PR DESCRIPTION
**Pull request by m00nbyte**
## Double check these details before you open a PR

- [x] PR does not match another non-stale PR currently opened
- [x] PR name matches the format *new icon: <i>Icon name</i> (<i>versions separated by comma</i>)*. More details [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#overview)
- [x] PR's base is the `develop` branch.
- [x] Your icons are inside a folder as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#organizational-guidelines)
- [x] SVG matches the standards laid out [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#svgStandards)
- [x] A new object is added in the `devicon.json` file as seen [here](https://github.com/devicons/devicon/blob/develop/CONTRIBUTING.md#-updating-the-deviconjson-)

## Link to prove your SVG is correct and up-to-date.
- [NPM Website](https://www.npmjs.com/)

### CODE REVIEW

File: icons/npm/npm-plain.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 128 128"><rect fill="#cc3534" x="4.83" y="4.83" width="118.35" height="118.35" rx="6.53" ry="6.53" /><path fill="#fff" d="M27.934 101.526h36.141l.046-54.119h18.024l-.046 54.165h18.07l.047-72.189-72.19-.092-.092 72.235z" /></svg>
```
Comment (Snailedlt) [inline icons/npm/npm-plain.svg:1]: This file should be renamed to original. It's multi-colored so it does not fit the criteria for plain versions, you can read more about the requirements and guidelines for each version here: https://github.com/devicons/devicon/wiki/SVG-Versions

**Review by Snailedlt — CHANGES_REQUESTED** [review]
This is a great addition!
We just need to change the name of the svg. Remember to also change the version name in `devicon.json`.

If you want to make a plain version, you can just clip out the white part, and make sure the end result is a single path without any clipPath or other advanced objects.

If you want any help, let me know :)

File: icons/npm/npm-plain.svg (hunk 1 above)
Comment (m00nbyte) [reply icons/npm/npm-plain.svg]: Thanks for your review, this is my first one :)

I renamed the file and updated and the JSON. I also added a new plain version.
Let me know if i made this right.

**Review by Snailedlt — CHANGES_REQUESTED** [review]
Great work so far!
There's a few minor issues, and a small suggestion I'd like to make:
1. The colors on these versions are different from the one already in the repo. NPM is very inconsistent in their branding when it comes to colors, I checked three different official logos and they all had different shades of gray... so for now I think we should just stick to the one that's already in the repo. That is `#cc3534`.
2. The icons should be resized to fit the entire viewbox. You can see how to do that in Inkscape [here](https://github.com/devicons/devicon/issues/1306#issuecomment-1788102718) or [here](https://github.com/devicons/devicon/issues/1306#issuecomment-1296051881)
3. The current original-wordmark has transparent text, while the official wordmark version has white text. Notice the white text in the comparison below

   | our version | official version |
   | ------------ | ---------------- |
   |   | npm-logo |


   We should add this official version, and name it `npm-original-wordmark`, and rename 
   the current original-wordmark to `npm-plain-wordmark`. That way we have both the 
   original version, and the font supported plain version.

Hopefully that was all understandable, if not let me know! :)

File: icons/npm/npm-plain.svg (hunk 1 above)
Comment (m00nbyte) [reply icons/npm/npm-plain.svg]: I resized the icons and changed the color.

**Review by canaleal — CHANGES_REQUESTED** [review]
Good work so far 👍 

Unfortunately though, the `npm-original-wordmark.svg` was removed and the `original-plain-wordmark` does not exist.

**Comment by m00nbyte** [comment]
Thanks, but i'm not sure what you mean.

i can see the logo being used here


and here

**Comment by canaleal** [comment]
@m00nbyte The `original-wordmark.svg` still exists in the versions, but there is no `original-wordmark.svg` file. 
image

Also, although npm uses the square version of the logo, they also still make use of the wordmark version as seen on their [website](https://www.npmjs.com/) and is found in this [archived repo](https://github.com/npm/logos).

**Comment by m00nbyte** [comment]
@canaleal this was already in the devicon.json before my PR and i can see that this file exists: https://github.com/devicons/devicon/blob/master/icons/npm/npm-original-wordmark.svg
should i remove the file and the entry from the json? i'm not sure what this have to do with my PR.

**Comment by canaleal** [comment]
@m00nbyte Correct, what I meant to say is that our version is not correct and should be changed. As @Snailedlt mentioned above, our version has transparent text while the original has white text.

**Comment by m00nbyte** [comment]
@canaleal ok i see, are those two icons i've added correct?

**Comment by canaleal** [comment]
@m00nbyte Both icons look nearly perfect. 

They're scaled up a little too much with a height and width of 128.005
image

And the plain icon also has a stroke width. 
image

Aside from those 2 things, they're good.

**Comment by m00nbyte** [comment]
@canaleal i removed the stroke-width but the size of the icons are 128px.
i looked at them in multiple programs and the viewbox is also limited to 128px so i have no idea how you get 128.005px.

**Review by canaleal — APPROVED** [review]
Excellent 👍

**Review by Snailedlt — APPROVED** [review]

**Pull request merged**